src/election_anomaly/db_routines/__init__.py

```diff
@@ -359,9 +363,9 @@ def dframe_to_sql(dframe,session,table,index_col='Id',flush=True,raw_to_votecoun
         id_enhanced_dframe = dframe.merge(
             up_to_date_dframe,left_on=intersection_cols,right_on=intersection_cols,how='inner').drop(
             target_only_cols,axis=1)
-        return id_enhanced_dframe, error
+        return id_enhanced_dframe, error_string
```

Comment:
Minor point here, but it looks like we could simply overwrite `up_to_date_dframe` in row 363 (`up_to_date_dframe = dframe.merge(  up_to_date_dframe,left_on=intersection_cols,right_on=intersection_cols,how='inner').drop(up_to_date_dframe,left_on=intersection_cols...`. This means we could always `return up_to_date_dframe, error_string` instead of wrapping the return in an if/else block.